<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>profileWindow</class>
 <widget class="QMainWindow" name="profileWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>660</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>660</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>660</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::TabShape::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>481</width>
      <height>641</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Profiles</string>
        </attribute>
        <widget class="QGroupBox" name="profileBox">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>0</y>
           <width>451</width>
           <height>331</height>
          </rect>
         </property>
         <property name="title">
          <string>Profiles</string>
         </property>
         <widget class="QWidget" name="horizontalLayoutWidget_2">
          <property name="geometry">
           <rect>
            <x>33</x>
            <y>290</y>
            <width>391</width>
            <height>26</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="profileButtons">
           <item>
            <widget class="QPushButton" name="newProfileButton">
             <property name="text">
              <string>New Profile</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="editProfileButton">
             <property name="text">
              <string>Edit Profile</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="copyProfileButton">
             <property name="text">
              <string>Copy Profile</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="deleteProfileButton">
             <property name="text">
              <string>Delete Profile</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QPlainTextEdit" name="descriptionBox">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>170</y>
            <width>431</width>
            <height>101</height>
           </rect>
          </property>
          <property name="focusPolicy">
           <enum>Qt::FocusPolicy::NoFocus</enum>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="plainText">
           <string/>
          </property>
         </widget>
         <widget class="QListWidget" name="listWidget">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>20</y>
            <width>411</width>
            <height>141</height>
           </rect>
          </property>
         </widget>
        </widget>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>340</y>
           <width>451</width>
           <height>251</height>
          </rect>
         </property>
         <property name="title">
          <string>Default Profile </string>
         </property>
         <widget class="QPlainTextEdit" name="plainTextEdit_3">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>50</y>
            <width>171</width>
            <height>91</height>
           </rect>
          </property>
          <property name="plainText">
           <string>Acceleration Y
time_elapsed</string>
          </property>
         </widget>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>20</y>
            <width>81</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Key Attributes</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>230</x>
            <y>10</y>
            <width>81</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Extra Attributes</string>
          </property>
         </widget>
         <widget class="QPlainTextEdit" name="plainTextEdit_4">
          <property name="geometry">
           <rect>
            <x>230</x>
            <y>50</y>
            <width>171</width>
            <height>91</height>
           </rect>
          </property>
          <property name="plainText">
           <string>Acceleration X
Acceleration Z
gyro x,y,z
rssi
etc.</string>
          </property>
         </widget>
         <widget class="QWidget" name="verticalLayoutWidget">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>150</y>
            <width>381</width>
            <height>91</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QPushButton" name="pushButton_6">
             <property name="text">
              <string>Rename Profile</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_5">
             <property name="text">
              <string>Clear All Csvs</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Tab 2</string>
        </attribute>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
